quipucords/scanner/network/connect.py

```diff
@@ -200,18 +208,22 @@ def run_with_result_store(self, result_store):
 
 
 # pylint: disable=too-many-arguments
-def connect(hosts, callback, credential, connection_port, forks=50):
+# pylint: disable=dangerous-default-value
+def connect(hosts, callback, credential, connection_port, forks=50,
```

Comment:
Probably should use None and not [].  Also remove the disable=dangerous-default-value